package.json

```diff
@@ -40,7 +40,6 @@
     "less": "~2.7.1",
```

Comment:
The permission(mode) of this file seems to be changed. Note that the executable bit differences can be ignored by `git config core.fileMode false`:
```
core.fileMode
       If false, the executable bit differences between the index and the
       working copy are ignored; useful on broken filesystems like FAT.
       See git-update-index(1). True by default.
```
(from `git-config(1)`)

Reply:
Thanks for the tip!  I'll do `git config --global core.fileMode false` on my Windows machines.

Both comments should be addressed now.